setup.py

```diff
@@ -129,6 +129,9 @@ def run_tests(self):
         'h.worker': [
             'notification=h.notification.worker:run',
             'nipsa=h.api.nipsa.worker:worker',
+        ],
+        'h.annotool': [
+            'reprepare=h.api.search:prepare',
```

Comment:
Hehe. Can we just call this `prepare` instead of `reprepare`? After all, the first time we run this things won't even have been `prepare`d.